crates/bitwarden-crypto/src/cose.rs

```diff
@@ -0,0 +1,8 @@
+//! This file contains private-use constants for COSE encoded key types and algorithms.
+//! Standardized values from <https://www.iana.org/assignments/cose/cose.xhtml> should always be preferred
+//! unless there is a specific reason to use a private-use value.
+
+// XChaCha20 <https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-xchacha-03> is used over ChaCha20
+// to be able to randomly generate nonces, and to not have to worry about key wearout. Since
+// the draft was never published as an RFC, we use a private-use value for the algorithm.
+pub(crate) const XCHACHA20_POLY1305: i64 = -70000;
```

Comment:
Question: Assuming there is ever a standarized value for this. Would we migrate over? What would that migration look like?